ci_framework/playbooks/06-deploy-edpm.yml

```diff
@@ -4,30 +4,39 @@
     step: pre_deploy
   ansible.builtin.import_playbook: ./hooks.yml
 
-- name: Prepare EDPM baremetal playbook
+- name: Deploy podified control plane
   hosts: "{{ cifmw_target_host | default('localhost') }}"
   gather_facts: false
   tasks:
     - name: Load parameters files
       ansible.builtin.include_vars:
         dir: "{{ cifmw_basedir }}/artifacts/parameters"
 
-    - name: Deploy podified control plane
+    - name: Run edpm_prepare
       ansible.builtin.import_role:
         name: edpm_prepare
 
-    - name: Perform Podified and EDPM deployment on compute nodes with virtual baremetal
-      when: cifmw_edpm_deploy_baremetal | default('false') | bool
-      ansible.builtin.import_role:
-        name: edpm_deploy_baremetal
-
 - name: Run post_ctlplane_deploy hooks
-  when: not cifmw_edpm_deploy_baremetal | default('false') | bool
```

Comment:
so this is to be run on all kind of env - baremetal and pre-provisionned?

Reply:
Why not? That's been defined for multinode jobs too[1]. It should also run validate_podified_deployment.yml like other jobs.

[1] https://github.com/openstack-k8s-operators/ci-framework/blob/main/scenarios/centos-9/multinode-ci.yml#L13